### PR TITLE
Update dependency on elastic-package

### PR DESCRIPTION
**Pull request by mtojek**
This PR updates dependency on elastic-package to include latest changes with multiple variants support.

### CODE REVIEW

**Comment by mtojek** [comment]
Ok, it seems that updating service variants went well for most of integrations, something is wrong with mysql integrations (FYI @jsoriano). I will investigate what's wrong with it.

**Comment by mtojek** [comment]
Ok, I updated the Docker image for MySQL to dynamically create `.cnf` file for error logs. I've changed also healthcheck to collect more error entries in the runtime:

```
2021-07-12T12:45:37.400625Z 40 [Note] Access denied for user 'root'@'localhost' (using password: YES)
2021-07-12T12:45:38.481326Z 42 [Note] Access denied for user 'root'@'localhost' (using password: YES)
2021-07-12T12:45:39.570612Z 44 [Note] Access denied for user 'root'@'localhost' (using password: YES)
2021-07-12T12:45:40.656779Z 46 [Note] Access denied for user 'root'@'localhost' (using password: YES)
2021-07-12T12:45:41.746385Z 48 [Note] Access denied for user 'root'@'localhost' (using password: YES)
2021-07-12T12:45:42.826265Z 50 [Note] Access denied for user 'root'@'localhost' (using password: YES)
2021-07-12T12:45:43.905627Z 52 [Note] Access denied for user 'root'@'localhost' (using password: YES)
```

**Review by jsoriano — APPROVED** [review]
Nice! :+1:

File: packages/mysql/_dev/deploy/docker/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Simulate hacking attempt to put some entries in the error log
+mysql -u root -pbad_password -hlocalhost -P 3306 > /dev/null
```
[the remark this answers is not in the document]
Comment (mtojek) [reply packages/mysql/_dev/deploy/docker/healthcheck.sh:4]: Frankly speaking this one truly solved the problem related to https://github.com/elastic/elastic-package/issues/405 . Without this change I was able to run successfully all system tests locally, but failed all CI runs.

**Pull request merged**